### DISCUSSION TITLE
Solve the problem of SyntaxError: Invalid or unexpected token

### DISCUSSION THREAD

**Discussion by xiaoxiaojx** [vercel/next.js#13828] [Ideas]
### Problem description:
> When CSS and other files are import in node_modules, compilation will throw error

### Problem analysis:
> The root cause is node environment when SSR rendering, node  cannot process these types of files

### Problem solving: 
> This package can ignore CSS and other files when multi-threaded packaging
https://github.com/xiaoxiaojx/with-ssr-entry
> If there are font files, PR can be raised

**Comment by xiaoxiaojx** [comment]
![Screenshot_20200529-221454_抖音短视频](https://user-images.githubusercontent.com/23253540/84097894-6655a200-aa38-11ea-8283-aa9da70daaba.jpg)

**Comment by jackywq** [comment]
> ### Problem description:
> > When CSS and other files are import in node_modules, compilation will throw error
> 
> ### Problem analysis:
> > The root cause is node environment when SSR rendering, node  cannot process these types of files
> 
> ### Problem solving:
> > This package can ignore CSS and other files when multi-threaded packaging
> > https://github.com/xiaoxiaojx/with-ssr-entry
> > If there are font files, PR can be raised

It doesn't work when use `with-ssr-entry`, any other questions?
